cmd/subnet-utils.go

```diff
@@ -174,8 +175,63 @@ func getSubnetClient() *http.Client {
 	return client
 }
 
-func subnetHTTPDo(req *http.Request) (*http.Response, error) {
-	return getSubnetClient().Do(req)
+func subnetHTTPDo(req *http.Request) (resp *http.Response, err error) {
+	resp, err = getSubnetClient().Do(req)
+	if err == nil && globalDebug {
+		dumpHTTPReq(req, resp)
+	}
+	return
+}
+
+// dumpHTTP - dump HTTP request and response.
```

Comment:
`%s/dumpHTTP/dumpHTTPReq/g` or actuallythe function itself can be called `dumpHTTP` as it dumps both request and response.

Reply:
just saw this; I will fix this in another PR